Dependencies/Program.cs

```diff
@@ -632,6 +632,7 @@ public static void DumpUsage()
                 "Options :",
                 "  -h -help : display this help",
                 "  -json : activate json output.",
+		"  -depth : limit recursion depth when analyisng loaded modules or dependency chain. Default value is infinite.",
```

Comment:
Found a typo ("analy**is**ng"). Furthermore, you use tabs for indentation instead of spaces like the other lines.

Reply:
It will be fixed in the other pr.